orbit/cmd/fleetd_tables/fleetd_tables.go

```diff
@@ -16,6 +16,8 @@ var (
 	socket   = flag.String("socket", "", "Path to the extensions UNIX domain socket")
 	timeout  = flag.Int("timeout", 3, "Seconds to wait for autoloaded extensions")
 	interval = flag.Int("interval", 3, "Seconds delay between connectivity checks")
+	// verbose must be set because osqueryd will set it on the extension when running in verbose mode.
+	_ = flag.Bool("verbose", false, "Enable verbose informational messages")
```

Comment:
nit: Define this inside `main` to make the intention clearer.

Reply:
Wanted to keep all flags grouped (for readability).